tools/classes.js

```diff
@@ -623,7 +623,10 @@ var classes = [
   'foam.nanos.approval.FulfilledApprovablePredicate',
   'foam.nanos.approval.FulfilledApprovableRule',
   'foam.nanos.approval.AuthenticatedApprovalDAOAuthorizer',
-  'foam.nanos.approval.PopulateApprovalRequestsDAO'
+  'foam.nanos.approval.PopulateApprovalRequestsDAO',
+
+  // holders
```

Comment:
It isn't really a holder no. More like StringArray. Just remove this line.

Reply:
Done.